tests/test_relations.py

```diff
@@ -98,8 +99,23 @@ def test_from_name(self, _find_impl, join, relation_to_role_and_interface):
         self.assertEqual(res.conversations(), ['conv.join'])
 
     @mock.patch.dict('sys.modules')
+    @mock.patch.object(relations.Conversation, 'join')
+    @mock.patch.object(relations, 'hookenv')
+    def test_cold_import(self, hookenv, conv_join):
+        tests_dir = os.path.dirname(__file__)
+        hookenv.charm_dir.return_value = os.path.join(tests_dir, 'data')
+        sys.modules.pop('relations.hyphen-ated.peer', None)
```

Comment:
Thats an odd way of spelling " del sys.modules['relations.hyphen-ated.peer'] ". I thought there was something magic happening here for a moment (is there?)